pages/index.tsx

```diff
@@ -11,6 +11,8 @@ import Head from "next/head"
 
 interface IndexProps {}
 
+
```

Comment:
extra empty line

Reply:
added the robots.txt there now.

Reply:
i'll remove the extra lines, i was trying to do something there on state-management, but i got it wrong.